relayer/src/chain/cosmos.rs

```diff
@@ -112,12 +115,32 @@ pub struct CosmosSdkChain {
     tx_config: TxConfig,
     rpc_client: HttpClient,
     grpc_addr: Uri,
+    light_client: TmLightClient,
     rt: Arc<TokioRuntime>,
     keybase: KeyRing,
     /// A cached copy of the account information
     account: Option<Account>,
 }
 
+async fn init_light_client(
```

Comment:
Nit: can we move the async function to the bottom of the file together with other async helper functions?